schemas/preferences.sublime-settings.json

```diff
@@ -406,19 +406,60 @@
           "oneOf": [
             {
               "type": "array",
               "uniqueItems": true,
               "items": {
                 "enum": [
                   "selection",
+                  "selection_none",
+                  "selection_tabs",
+                  "selection_spaces",
+                  "selection_mixed",
+                  "selection_mixed_tabs",
+                  "selection_mixed_spaces",
+                  "selection_all",
                   "leading",
+                  "leading_none",
+                  "leading_tabs",
+                  "leading_spaces",
+                  "leading_mixed",
+                  "leading_mixed_tabs",
+                  "leading_mixed_spaces",
+                  "leading_all",
                   "enclosed",
+                  "enclosed_none",
+                  "enclosed_tabs",
+                  "enclosed_spaces",
+                  "enclosed_mixed",
+                  "enclosed_mixed_tabs",
+                  "enclosed_mixed_spaces",
+                  "enclosed_all",
                   "trailing",
+                  "trailing_none",
+                  "trailing_tabs",
+                  "trailing_spaces",
+                  "trailing_mixed",
+                  "trailing_mixed_tabs",
+                  "trailing_mixed_spaces",
+                  "trailing_all",
                   "isolated",
-                  "all"
+                  "isolated_none",
+                  "isolated_tabs",
+                  "isolated_spaces",
+                  "isolated_mixed",
+                  "isolated_mixed_tabs",
+                  "isolated_mixed_spaces",
+                  "isolated_all",
+                  "all",
+                  "all_none",
+                  "all_tabs",
+                  "all_spaces",
+                  "all_mixed",
+                  "all_mixed_tabs",
+                  "all_mixed_spaces",
```

Comment:
Around this point JSON server reports a parsing error. Will investigate what is that about...

![Screenshot 2020-09-19 at 20 08 18](https://user-images.githubusercontent.com/153197/93686052-f84b5d00-fab3-11ea-8d9c-1deea4c6a673.png)

Reply:
Looks like it's the trailing comma.
Hmm, that shouldn't be a problem...

Reply:
Not sure why `allowTrailingCommas` from sublime-base schema is not respected so lets just remove those from `sublime-base.json`:
```
  "allowComments": true,
  "allowTrailingCommas": true,
```
Then we'll get validation errors.

Reply:
I guess that's why we should always care about trailing commas even if ST is permissive with them.

Reply:
The settings are about how to validate a _Preferences.sublime-settings_ file. They don't control how the json-schema file itself is parsed.

Reply:
I'm talking about `allowTrailingCommas` in `sublime-base.json`. That one is used to validate this (and other) schemas since it's used as a base of this schema. And it works since the editor doesn't show you errors related to trailing commas. The problem is that while it works when validating other json files, it doesn't apply when parsing the schema itself so it's a bit inconsistent in how JSON server handles it.

Can you please do the change I've requested? It's the right thing to do as then the results will be consistent with what you see in the editor and what the server expects to parse the schema correctly.

Reply:
Remove those `allowTrailingCommas`, ... from the base-scheme? Yes, can do.